server/conf/docker/docker-compose-dev.all.debug.yml

```diff
@@ -61,7 +61,7 @@ services:
       - PGDATA=/var/lib/postgresql/data/portal
 
   postgrescms:
-    image: postgres:11.5
+    image: postgres:12.0
     volumes:
```

Comment:
Django version 4 dropped support for potgres version 11 so this was upgraded. My question is that should this be upgraded to 12.0 (which will likely drop support in Nov 2024) or upgrade to the latest version which is version 16. Upgrading to the latest version might require more changes on our end that I haven't had the chance to look into it. See postgres versions [here](https://www.postgresql.org/support/versioning/)

Reply:
The production Postgres instance `wma-postgres-01` runs Postgres v14.9, so I'd recommend using that version.